plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/views/qm/QueryManagerViewFilter.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ui.views.qm;
+
+import org.eclipse.swt.widgets.Composite;
+import org.jkiss.code.NotNull;
+import org.jkiss.dbeaver.ui.controls.querylog.QueryLogViewer;
+
+public interface QueryManagerViewFilter {
```

Comment:
I'm not sure there will ever be more than one implementation of this interface. Do you have some other possible implementations in your mind?

Reply:
The implementation is unavailable in CE, so it must be wired somehow. I haven't found a suitable interface among existing ones (besides `IObjectPropertyConfigurator`, but it offers too many things I'm not interested in)